src/locale/keymap-util.c

```diff
@@ -780,3 +782,141 @@ int x11_convert_to_vconsole(Context *c) {
 
         return modified;
 }
+
+static
+bool locale_encoding_is_utf8_or_unspecified(const char *locale) {
```

Comment:
our usual coding style would place the `static` on the same line as the the rest of the function prototype. hence, please remove one linebreak here